src/ui/views/components/window/Window.tsx

```diff
@@ -1,22 +1,15 @@
 import { ReactNode } from "react";
 import classNames from "classnames";
 
-import { Content } from "../../../atoms/content/Content";
 import "./Window.css";
 
 interface WindowProps {
   className?: string;
-  header: ReactNode;
-  footer?: ReactNode;
   children: ReactNode;
 }
 
-export function Window({ className, header, footer, children }: WindowProps) {
-  const windowClass = classNames("Window", className);
+export function Window({ className, children }: WindowProps) {
+  const windowClass = classNames("Window flex", className);
```

Comment:
I usually just put these inline with the JSX unless it makes it super unreadable.